catalogue/webapp/pages/work.js

```diff
@@ -92,20 +92,20 @@ export const WorkPage = ({
           <div className='container'>
             <div className='grid'>
               <div className={grid({s: 12})}>
-                <SecondaryLink
-                  link={worksUrl({query, page})}
-                  text='Search results'
-                  trackingEvent={{
-                    category: 'component',
-                    action: 'return-to-results:click',
-
-                    label: `id:${work.id}, query:${query || ''}, title:${work.title}`
+                <NextLink {...worksUrl({query, page})}>
+                  <a onClick={() => {
+                    trackEventV2({
```

Comment:
This isn't actually a `NextLink`, it is just a link.
Should this link fit into one of the other links styles and we track it like that?

Reply:
(basically this would be the only thing we track as next link, which in itself isn't a UI component, so feels odd)

Reply:
If it is a snow flake, maybe we call it by what it is e.g. `BackToResults`?